src/selectors/layout.test.js

```diff
@@ -23,7 +23,8 @@ describe('Selectors', () => {
     const prepareLargeDataset = () => {
       const data = { ...animals };
       let extraNodes = [];
-      const iterations = Math.ceil(largeGraphThreshold / data.nodes.length) + 1;
+      const iterations =
+        Math.ceil((largeGraphThreshold * 2) / data.nodes.length) + 1;
```

Comment:
Instead of just doubling the threshold and adding a comment, I'd prefer for us to get the number of actual visible nodes. You could do this with a selector:
```suggestion
      const visibleNodeCount = getVisibleNodeIDs(mockState.animals).length;
      const iterations = Math.ceil((largeGraphThreshold) / visibleNodeCount) + 1;
```